### PR TITLE
Add eva density plot

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
@danholdaway added a density plot option on `eva` that was not added to `EMCPy` in the transfer of code. Re-introducing that option now. Did not include line at zero because that can be added to the plot as a separate layer in the subplot.

### CODE REVIEW

File: src/emcpy/plots/create_plots.py [hunk 1]
```diff
@@ -407,7 +408,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             c=plotobj.data, s=plotobj.markersize,
-                            **inputs, norm=norm, transform=self.projection.projection)
+                            **inputs, norm=norm, transform=ccrs.PlateCarree())
```
Comment (kevindougherty-noaa) [inline src/emcpy/plots/create_plots.py]: I believe this was from a different change that got mixed with this PR. This needs to be changed though and I can show reasons in a future PR.

Comment (CoryMartin-NOAA) [reply src/emcpy/plots/create_plots.py]: This looks like the thing that Jakir was working on for polar projections. It needs to support multiple projections.

Comment (kevindougherty-noaa) [reply src/emcpy/plots/create_plots.py]: I'll revert the changes in this PR. However, I will show issues it creates I found when adding new features in a separate PR.

**Review by danholdaway — APPROVED** [review]
Thanks for adding this upstream.

**Pull request merged**